local-cli/util/Config.js

```diff
@@ -25,26 +25,35 @@ let cachedConfig = null;
  * error will be thrown.
  */
 const Config = {
-  get(cwd, defaultConfig) {
-    if (cachedConfig) {
```

Comment:
In OSS at least, there's only one caller:
```
$ git grep 'Config.get('
local-cli/cliEntry.js:  command[0](args, Config.get(__dirname, defaultConfig)).done();
```

And since cliEntry.js runs just once, I don't think caching mattered. Also the cache doesn't create any cache key based on cwd + defaultConfig, so if you called Cache.get with different arguments you might get the wrong cached value.